src/main/java/org/datadog/jmxfetch/Connection.java

```diff
@@ -68,7 +61,7 @@ protected void createConnection() throws IOException {
         this.env.put("attribute.remote.x.request.waiting.timeout", CONNECTION_TIMEOUT);
```

Comment:
I tried to find out where this property might be used. On the "official" oracle side I found something similar only in the the (jmx spec)[https://docs.oracle.com/javase/8/docs/technotes/guides/jmx/JMX_1_4_specification.pdf] with a minor variation (`jmx.remore.x.request.waiting.timeout` listed as a non-standard parameter). So far the only implementation I found that seems to honour this parameter is (weblogic)[https://docs.oracle.com/cd/E13222_01/wls/docs103/jmx/accessWLS.html#wp1112969].